environment.yml

```diff
@@ -1,5 +1,10 @@
 name: netket_env
 
+channels:
+  - defaults
+  - pytorch 
+  - conda-forge
```

Comment:
is conda-forge needed?

Reply:
Actually, you don't need any extra channels. `defaults` suffices.

Reply:
Not sure that on Mac defaults is enough

Reply:
I think when you use `defaults` and ask for `pytorch`, the following package gets installed: https://anaconda.org/anaconda/pytorch (I just compared the hashes locally). And it has versions for Linux, OS X & Windows. As a Mac owner you could check ;) but I do think `defaults` should do the trick.